core/src/main/java/org/elasticsearch/index/Index.java

```diff
@@ -31,21 +32,23 @@
 public class Index implements Streamable {
 
     private String name;
```

Comment:
sure - wanted to keep this as small as possible (it's big enough :)). I'll tackle writable too.